tools/debug-ui/src/BlocksView.tsx

```diff
@@ -0,0 +1,188 @@
+import './BlocksView.scss';
+import { useQuery } from 'react-query';
+import { fetchChainProcessingStatus, BlockProcessingStatus, ChunkProcessingStatus } from './api';
+
+type BlocksViewProps = {
+    addr: string;
+};
+
+function prettyTime(datetime: string) {
+    const time = new Date(Date.parse(datetime));
+    return (
+        time.getUTCHours() +
+        ':' +
+        String(time.getUTCMinutes()).padStart(2, '0') +
+        ':' +
+        String(time.getUTCSeconds()).padStart(2, '0') +
+        '.' +
+        String(time.getUTCMilliseconds()).padStart(3, '0')
+    );
+}
+
+function printTimeInMs(time: number | null) {
```

Comment:
nit: return type

Reply:
nit: does tsx work with number?

Reply:
tsx is just the react syntax, like `<element attribute="value">contents</element>`. Everything else works just like ts. The return type here should just be annotated as string.

Reply:
added return type